client/src/components/Navbar/navbar.jsx

```diff
@@ -162,27 +175,25 @@ const Navbar = () => {
                     </Link>
                 </div>
                 <div
-                    className={`flex w-[30%] justify-center gap-4 max-lg:w-full max-lg:justify-end ${
-                        user ? "hidden" : ""
-                    }`}
+                    className={`flex w-[30%] justify-center gap-4 max-lg:w-full max-lg:justify-end ${user ? "hidden" : ""
+                        }`}
                 >
                     <Link to="login">
                         {" "}
-                        <button className="inline-block rounded-full border-2 border-neutral-800 px-4 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-neutral-800 transition duration-150 ease-in-out hover:border-neutral-800 hover:bg-black hover:text-white focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 max-lg:hidden">
+                        <button className="inline-block rounded-full border-2 border-neutral-800 px-4 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-neutral-800 transition duration-100 ease-in-out hover:border-neutral-800 hover:bg-black hover:text-white focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 max-lg:hidden hover:text-white">
                             Login
                         </button>{" "}
                     </Link>
                     <Link to="signup">
                         {" "}
-                        <button className="inline-block bg-black  rounded-full border-2 border-neutral-800 px-6 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-white transition duration-150 ease-in-out hover:border-neutral-800 hover:bg-white hover:text-black focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 ">
+                        <button className="inline-block bg-black  rounded-full border-2 border-neutral-800 px-6 pb-[6px] pt-2 text-[12px] font-medium uppercase leading-normal text-white transition duration-100 ease-in-out hover:border-neutral-800 hover:bg-white hover:text-black focus:border-neutral-800 focus:text-neutral-800 focus:outline-none focus:ring-0 active:border-neutral-900 active:text-neutral-900 focus:text-white">
```

Comment:
There are several other instances in the code that use the same class components and have the same clickable error. If you think it's appropriate, we can address their fixes in this PR. WDYT?

Reply:
> There are several other instances in the code that use the same class components and have the same clickable error. If you think it's appropriate, we can address their fixes in this PR. WDYT?

Yes I guess!

Reply:
Sounds great. Ping us when you are done with the changes.

CC @ShashankShekhar07

Reply:
@dvjsharma It's done!